parser-typechecker/src/Unison/TypePrinter.hs

```diff
@@ -143,7 +146,13 @@ prettySignatures'
   :: Var v => PrettyPrintEnv
   -> [(HashQualified Name, Type v a)]
   -> [Pretty ColorText]
-prettySignatures' env ts = map PP.syntaxToColor $ PP.align
+prettySignatures' env ts = map PP.syntaxToColor $ prettySignatures'' env ts
+
+prettySignatures''
+  :: Var v => PrettyPrintEnv
+  -> [(HashQualified Name, Type v a)]
+  -> [Pretty SyntaxText]
```

Comment:
Same deal here - needed a version that returned `Pretty SyntaxText` for doc rendering in codebase server.